Bayes_Rules/Chapter 7.ipynb

```diff
@@ -0,0 +1,1302 @@
+{
```

Comment:
<p>Add watermark to the bottom</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-resources/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Added